src/udev/udevadm-query.c

```diff
@@ -0,0 +1,148 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
```

Comment:
Doesn't belong to this PR. Rebase failure?

Reply:
Consider it a sneak peek ;)